src/alliteratedActors.js

```diff
@@ -22,7 +22,12 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  const regex = /\b[A-Za-z]/g;
+
+  return names.find(name =>
+    (name.match(regex)).every(letter => {
```

Comment:
You have an interesting approach. I wonder if we could solve this task with `regex.test()`